glean-core/ffi/src/weak.rs

```diff
@@ -0,0 +1,57 @@
+//! Based on `library/std/src/sys/unix/weak.rs` from the Rust libstd code base.
```

Comment:
Should we note down the revision we pulled this from?

Reply:
That file upstream is essentially unchanged since 2016. I can add a Rust version number maybe.